src/indexer/merger.rs

```diff
@@ -441,20 +426,17 @@ impl IndexMerger {
             max_term_ords.push(terms.num_terms() as u64);
         }
 
-        let mut term_ord_mapping_opt =
-            if *field_type == FieldType::HierarchicalFacet {
-                Some(TermOrdinalMapping::new(max_term_ords))
-            } else {
-                None
-            };
-
+        let mut term_ord_mapping_opt = if *field_type == FieldType::HierarchicalFacet {
```

Comment:
I wonder if we can configure rustfmt to avoid this?

Reply:
Test has been added but I'm not super happy with it. 

1. Should I do this much setup, or is there an easier :tm: way?
2. Since I don't understand some of the internals of the meta, etc. I may be missing an edge case.